### ISSUE TITLE
Prophet integration

### ISSUE THREAD

**Issue by grigoryan-davit** [aimhubio/aim#2496]
## 🚀 Feature

Integration with Prophet (https://github.com/facebook/prophet).

### Motivation

Prophet is a widely used forecasting algorithm with an official implementation by Meta. While it is trained substantially differently from neural networks, it's still useful to log hyperparameters and relevant metrics during different experiments. 

### Pitch

* AimLogger object that by default only logs the hyperparameters of a Prophet model
* The user can optionally log any validation metric they want
